content/en/blog/2023/announcing-notation-rc3.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Notation v1.0.0-RC.3 is available!
+author:  "Notation Release Team"
+date: 2023-03-08
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation CLI v1.0.0-RC.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release introduces a breaking change to Notation. The default type of signature manifest is changed to image manifest. The flag `--signature-manifest` for `notation sign` command is experimental for users to store signatures using artifact manifest.
```

Comment:
It will be worth elaborating on this statement. This is not breaking the validation of the signatures - signatures that are saved with artifact manifest are still verifiable. Maybe it is worth to reword with something like this:

> This release introduces a change to how signatures are stored in the registries to align with the OCI direction to use image manifest. The default type of signature manifest is changed to image manifest. The flag `--signature-manifest` for `notation sign` command is experimental for users who want to store signatures using artifact manifest. This change does not impact the validation of signatures.